keyboards/ymdk/ymd60/readme.md

```diff
@@ -0,0 +1,26 @@
+# YMDK 60 HHKB
+
+![YMDK60 HHKB PCB](https://i.imgur.com/5lRZfcgh.png)
+
+A 60% PCB, which is controlled by an ATMega32U4 chipset.
+
+* Keyboard Maintainer: [Pangorin](https://github.com/pangorin)
+* Hardware Supported: ATMega32U4-MU
```

Comment:
```suggestion
* Hardware Supported: ATMega32U4-MU
* Hardware Availability: [YMDK](https://ymdkey.com/products/ymdk-hhkb-layout-qmk-via-programmable-underglow-rgb-aluminum-cnc-sandwich-case-plate-mechanical-keyboard-kit)
```

A link to the vendor's website is a good way to verify that the firmware is compatible with the hardware _(unless there's a good reason for omitting this bullet point from the template)_.

Reply:
Nice, I commited your suggestion!